pkgs/tools/text/mdbook-epub/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, bzip2
+, CoreServices
+}:
+
+let
+  pname = "mdbook-epub";
+  version = "unstable-2022-12-25";
+in rustPlatform.buildRustPackage {
+  inherit pname version;
+
```

Comment:
```suggestion
in rustPlatform.buildRustPackage rec {
  pname = "mdbook-epub";
  version = "unstable-2022-12-25";

```

Reply:
Yes it is, and we are currently implementing alternatives, https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes, but rec is fine in most cases.